### PR TITLE
GUI: Add icons package

### PR DESCRIPTION
**Pull request by dinuxlt**
This package contains two icons:
image
image

### CODE REVIEW

File: cmd/nerf/icons/icons.go [hunk 1]
```diff
@@ -0,0 +1,2299 @@
+//+build linux darwin
```
Comment (ton31337) [inline cmd/nerf/icons/icons.go:1]: Who uses this file?

Comment (ton31337) [reply cmd/nerf/icons/icons.go:1]: Is this needed for systray?

Comment (dinuxlt) [reply cmd/nerf/icons/icons.go:1]: Generated this file using tools from https://github.com/getlantern/systray/tree/master/example/icon
I think there is icons format difference between linux/osx and windows.

Later i will import this package and I will use this for systray.

Comment (ton31337) [reply cmd/nerf/icons/icons.go:1]: Do not forget to update instructions on how to build them.

**Review by ton31337 — APPROVED** [review]

**Pull request merged**